.github/copilot-instructions.md

```diff
@@ -0,0 +1,42 @@
+## Test Suggestions
+
+When converting a PR from WIP to publish, provide a concise test plan.
+
+- **Select Key Test Cases:** List only the essential test cases to validate the PR. Use pipe '|' to separate names (e.g., test1|test2|test3).
+- **List Impacted LISA Features:** Mention only the relevant LISA Features affected by this PR.
```

Comment:
LISA *Feature* is an object type in the LISA. So, this is not about LISA's feature, it's about "Feature" of test case supported.

Reply:
I think the updated content haven't explain the "Feature" class clear yet. Give a more comprehensive explanation of the concept to avoid confusion.